### ISSUE TITLE
Feature idea - Sort todos by drag&drop

### ISSUE THREAD

**Issue by DevHD83** [cassidoo/todometer#46]
It would be great if sorting would be supported.
I'm used to put the urgent todos at the top position – so it would be great if sorting by drag&drop could be implemented.

By the way: awesome app, thank you for sharing this!

**Comment by wuweiweiwu** [comment]
@cassidoo I was thinking React DnD. And if no one is taking it I do it :)

**Comment by Rohitkrops** [comment]
@wuweiweiwu [React beautiful dnd](https://github.com/atlassian/react-beautiful-dnd) would be cool.

**Comment by vdedios** [comment]
Thank you for this awesome tool!

I totally support this idea, it would be awesome to be able to drag and drop items so you can sort them as you need. Sometimes when you organize your day you just brainstorm all the tasks without thinking which one is best to start with. 

Another useful idea this would be being able to edit the text on existing blocks, should I open a new issue for this? Thank you!

**Comment by ansikun2** [comment]
yeah, best-looking and simplest To-Do meter, I use it every day. A sorting function would be super, so users can drag the most important things to the top. A sync to a phone app would also be swell, in a later version... thanks so much!
